test/passing/infix_bind.ml

```diff
@@ -0,0 +1,115 @@
+[@@@ocamlformat "break-infix-before-func=true"]
+
+;;
+f x
+>>= fun y ->
+g y
+>>= fun () ->
+f x >>= fun y -> g y >>= fun () -> f x >>= fun y -> g y >>= fun () -> y ()
+
+;;
+f x
+>>= function
+| A -> (
+    g y
+    >>= fun () ->
+    f x
+    >>= fun y ->
+    g y
+    >>= function x -> ( f x >>= fun y -> g y >>= function _ -> y () ) )
+
+[@@@ocamlformat "break-infix-before-func=false"]
+
+;;
+f x >>= fun y ->
+g y >>= fun () ->
+f x >>= fun y -> g y >>= fun () -> f x >>= fun y -> g y >>= fun () -> y ()
+
+;;
+f x >>= function
+| A -> (
+    g y >>= fun () ->
+    f x >>= fun y ->
+    g y >>= function
+    | x -> ( f x >>= fun y -> g y >>= function _ -> y () ) )
+
+;;
+eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee |> fun x -> x
+
+;;
+eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
+  |> fun xxxxxx xxxxxxxxxx xxxxxxxx xxxxxxxx -> x
+
+;;
+eeeeeeeeeeeee eeeeeeeeeeeeeeeeee eeeeeeeeeeeeeeeeeee eeeeeeeeeeeeeeeeeeee
+  eeeeeeeeeeee eeeeeeeeee |> fun x -> x
+
+;;
+eeeeeeeeeeeee eeeeeeeeeeeeeeeeee eeeeeeeeeeeeeeeeeee eeeeeeeeeeeeeeeeeeee
+  eeeeeeeeeeee eeeeeeeeee
+  |> fun xxxxxxxxxxxxx xxxxxxxxxxxxxx xxxxxxxxxxxxxx xxxxxxxxx ->
+    xxxxxxxxxxx xxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxx xxxxxxxxxxx
+
+;;
+eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee |> fun x -> x
+
+;;
+eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
+  |> fun xxxxxx xxxxxxxxxx xxxxxxxx xxxxxxxx -> x
+
+;;
+eeeeeeeeeeeee eeeeeeeeeeeeeeeeee |> fun xxxxxxxxx xxxxxxxxxxxxx -> x
+
+;;
+eeeeeeeeeeeee eeeeeeeeeeeeeeeeee eeeeeeeeeeeeeeeeeee
+  |> fun xxxxxxxx xxxxxxxxx xxxxxxxxxxxxx -> x
+
+;;
+eeeeeeeeeeeee eeeeeeeeeeeeeeeeee |> fun xxxxxxxxxxxxx ->
+    xxxxxxxx xxxxxxxxxx xxxxxxxxx xxxxxxxxxxxxx
+
+;;
+eeeeeeeeeeeee eeeeeeeeee
+  |> fun xxxxxxxxxxxxx xxxxxxxxxxxxxx xxxxxxxxxxxxxx xxxxxxxxx ->
+    xxxxxxxxxxx xxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxx xxxxxxxxxxx
```

Comment:
`ocp-indent` will format this as:

```ocaml
eeeeeeeeeeeee eeeeeeeeee
|> fun xxxxxxxxxxxxx xxxxxxxxxxxxxx xxxxxxxxxxxxxx xxxxxxxxx ->
xxxxxxxxxxx xxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxx xxxxxxxxxxx
```

Reply:
I disagree with the formatting of ocp-indent here, is this really the behavior we want?

Reply:
there is now way to "come back" one level up, here, so yes I think that makes sense.

Reply:
it's the formating that people are getting used to :p All these light sequence combinators are very similar to `;` which doesn't increase the indentation.